src/Cassandra.IntegrationTests/Linq/Structures/Band.cs

```diff
@@ -0,0 +1,30 @@
+﻿// //
```

Comment:
`//` appears twice (we can fix the resharper macro later, but the source code should be clean)